CHANGELOG.md

```diff
@@ -8,6 +8,16 @@
 - Rename `pdata.AttributeMap.Delete` to `pdata.AttributeMap.Remove` (#4914)
 - pdata: deprecate funcs working with InternalRep (#4957)
 
+### 🚩 Deprecations 🚩
```

Comment:
We tracked these in the "breaking change".

Reply:
But we shouldn't add deprecations to _Breaking changes_ in 1.0 modules going forward, right? Should we reconsider this practice then?

Reply:
Not sure we will deprecate that many things after, but indeed deprecation is not breaking change :D, but in our context deprecation is a preparation for a breaking change :))

I agree, but if we do this, let's create also a PR and move what is today in breaking change to deprecations when it makes sense.

Reply:
Sounds good.

Reply:
Submitted a PR https://github.com/open-telemetry/opentelemetry-collector/pull/5004